bitrepository-repository-settings-xsd/src/main/resources/xsd/RepositorySettings.xsd

```diff
@@ -212,15 +212,53 @@
           <xs:annotation>
             <xs:documentation>
               The timeout for the identification in milliseconds.
+              Deprecated for removal, but currently still mandatory for backward compatibility.
             </xs:documentation>
           </xs:annotation>
         </xs:element>
+        <xs:element name="IdentificationTimeoutDuration" minOccurs="0" maxOccurs="1">
+          <xs:annotation>
+            <xs:documentation>
+              The timeout for the identification.
+              If present, clients should prefer to respect this setting over IdentificationTimeout.
+              Non-negative XML schema durations consisting of hours, minutes and/or seconds are accepted.
+              The seconds may have up to 9 decimals of fraction.
+              Any client may support only coarser precision than 9 decimals.
+              For example PT2H (2 hours), PT3M (3 minutes), PT1H22M22.5S (1 hour 22 minutes 22.5 seconds).
+              The setting is not allowed to contain any minus sign (-PT2M is *forbidden*).
+            </xs:documentation>
+          </xs:annotation>
+          <xs:simpleType>
+            <xs:restriction base="xs:duration">
+              <xs:pattern value="PT(\d+H)?(\d+M)?(\d+(\.\d{1,9})?S)?" />
+            </xs:restriction>
+          </xs:simpleType>
+        </xs:element>
         <xs:element name="OperationTimeout" type="xs:nonNegativeInteger">
           <xs:annotation>
             <xs:documentation>
               The timeout for the actual operation in milliseconds.
+              Deprecated for removal, but currently still mandatory for backward compatibility.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="OperationTimeoutDuration" minOccurs="0" maxOccurs="1">
```

Comment:
I see there is a small change here: both of the old settings didn't have _minOccurs_ or _maxOccurs_ specified before, so both were expected to occur exactly once. Although we could let it be the job of the using clients to define defaults if the settings are not specified, I think it'd be best to require these to be defined.